src/DynamoCoreWpf/UI/Themes/Modern/MenuStyleDictionary.xaml

```diff
@@ -402,6 +402,98 @@
             </Style>
         </Style.Resources>
     </Style>
-    
-    
+
+    <!-- Extensions Menu -->
+    <Style x:Key="ExtensionsStyle" 
+           TargetType="{x:Type MenuItem}"
+           BasedOn="{StaticResource MenuItemStyle}">
+        <Style.Resources>
+            <ControlTemplate x:Key="Extensions" 
+                             TargetType="MenuItem">
+                <Border Name="Border"
+                        BorderBrush="Transparent"
+                        BorderThickness="1"
+                        VerticalAlignment="Stretch"
+                        Padding="8,0,8,0"
+                        Margin="6,0,7,0">
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="25" />
+                            <ColumnDefinition Width="*"/>
+                        </Grid.ColumnDefinitions>
+                        <ContentPresenter Name="Icon"
+                                          Grid.Column="0"
+                                          Margin="0,0,6,0"
+                                          VerticalAlignment="Center"
+                                          ContentSource="Icon" />
+                        <ContentPresenter Name="Content"
+                                          Margin="0"
+                                          Grid.Column="1"
+                                          VerticalAlignment="Center"
+                                          ContentSource="Header"
+                                          RecognizesAccessKey="True" />
+                        <Popup Name="Popup"
```

Comment:
What would this style refer to?

Reply:
As the changes for the icon only apply for the extensions menu I had to override the ControlTemplate for TopLevelHeader on it, and that ControlTemplate includes the Popup style configuration so I had to be duplicated. It is the same as the one on line 48